chart/kubeapps/templates/kubeappsapis/deployment.yaml

```diff
@@ -87,6 +87,7 @@ spec:
           env:
             - name: PORT
               value: {{ .Values.kubeappsapis.containerPort | quote }}
+            {{- if .Values.redis.enabled }}
```

Comment:
Haha, we ran into the same issue. I've added these changes at https://github.com/kubeapps/kubeapps/pull/3204/files

Reply:
Yeah, I noticed your commit, which was why I wondered if you'd missed my PR :) Thanks for the review!